server/src/main/scala/com/cloudera/livy/server/interactive/CreateInteractiveRequest.scala

```diff
@@ -36,5 +36,6 @@ class CreateInteractiveRequest {
   var queue: Option[String] = None
   var name: Option[String] = None
   var conf: Map[String, String] = Map()
+  var scalaVersion: Option[String] = None
```

Comment:
I understand that. But I think we don't have a use case that requires different Scala version on a per session basis. To make this work, admins have to carefully setup CLASSPATH to include both spark jars from both Scala versions. Unless a lot of users are asking for it, I would avoid supporting it.

In the future we might want to support different Spark version on a per session basis. It's a big item. Mind if we tackle it separately, and keep Scala version a global LivyConf?